lib/zendesk_apps_support/location.rb

```diff
@@ -0,0 +1,29 @@
+module ZendeskAppsSupport
+  module Location
+    # the numbers below match the enum values on the database, do not change them!
```

Comment:
uh, is this something we want to publically have in our codebase? (i do realise the irony of asking this question in a public repo)